### ISSUE TITLE
Refine/Streamline Program Overview section

### ISSUE THREAD

**Issue by NAretakis** [18F/cms-hitech-apd#545]
We want to streamline the Program Overview section of the app so that the HIT Narrative, HIE Narrative, and MMIS Narrative subsections are no longer sitting on their own. Instead, under the Overview section, we need to have three rich text boxes: 

1. HIT Overview
2. HIE Overview
3. MMIS Overview

With those added so that they exist within the "Overview" collapsible section, we can remove the stand-alone HIT Narrative, HIE Narrative, and MMIS narrative sections.

### This task is done when...
 - [ ] Stand-alone sections are removed, and Overview subsections are added
 - [ ] Help text is provided to clarify what goes into each section

**Comment by quarterback** [comment]
@NAretakis @lauraponce I removed the three subsections and integrated them. I didn't mock the text into these, I just wanted to show the experience so you could comment.
<img width="1271" alt="screenshot 2018-05-15 17 40 32" src="https://user-images.githubusercontent.com/4461062/40085034-2b0256b4-5867-11e8-984e-11a6c2a34804.png">
<img width="1126" alt="screenshot 2018-05-15 17 48 34" src="https://user-images.githubusercontent.com/4461062/40085368-314fa430-5868-11e8-9b06-6cd741da9391.png">

**Comment by quarterback** [comment]
I'll draft the text that should go in these sections in the existing template so we have it and we can just integrate it later. I imagine those subheaders for MMIS/HIT/etc. could probably be a H3 or something rather than small bold text and if we wanted instructions, that could be listed as well.

**Comment by NAretakis** [comment]
Yes, that looks right. But can we change to HIT Overview, HIE Overview, and MMIS Overview? Narrative isn't a term that we've used in any of our existing docs.
